website/docs/d/container_orchestrator_versions.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_orchestrator_versions"
+sidebar_current: "docs-azurerm-datasource-container-orchestrator-versions-x"
+description: |-
+  Gets available versions of container orchestrators.
+---
+
+# Data Source: azurerm_container_orchestrator_versions
+
+Use this data source to access information about available versions of orchestrators supported by the Azure Container services.
+
+## Example Usage
+
+```hcl
+data "azurerm_container_orchestrator_versions" "current" {
+  location = "westeurope"
```

Comment:
in the examples we tend to format this as:

```suggestion
  location = "West Europe"
```

(as we automatically transform this internally)